tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -72,10 +68,63 @@ def test_createami(region, os, instance, request, pcluster_config_reader, vpc_st
         any("packages.chef.io".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
     ).is_false()
     assert_that(
-        any("Thank you for installing Chef".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+        any(
+            "Thank you for installing Cinc Client".lower() in s.lower()
+            for s in pcluster_createami_result.stdout.split("\n")
+        )
     ).is_true()
     assert_that(
-        any("Starting Chef Client".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+        any("Starting Cinc Client".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
     ).is_true()
 
     assert_that(pcluster_createami_result.stdout).does_not_contain("No custom AMI created")
+
+
+@pytest.mark.dimensions("us-west-2", "c5.xlarge", "centos7", "*")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "ubuntu1804", "*")
+@pytest.mark.dimensions("eu-west-1", "m6g.xlarge", "alinux2", "*")
+def test_createami_post_install(
+    region, os, instance, test_datadir, request, pcluster_config_reader, vpc_stack, architecture
+):
+    """Test post install script and base AMI is ParallelCluster AMI"""
+    cluster_config = pcluster_config_reader()
+
+    # Get ParallelCluster AMI as base AMI
+    ami_type = "pcluster"
+    base_ami = retrieve_latest_ami(region, os, ami_type=ami_type, architecture=architecture)
+
+    # Networking
+    vpc_id = vpc_stack.cfn_outputs["VpcId"]
+    networking_args = ["--vpc-id", vpc_id, "--subnet-id", vpc_stack.cfn_outputs["PublicSubnetId"]]
+
+    # Custom Cookbook
+    custom_cookbook = request.config.getoption("createami_custom_chef_cookbook")
+    custom_cookbook_args = [] if not custom_cookbook else ["-cc", custom_cookbook]
+
+    # Instance type
+    pcluster_version_result = run_command(["pcluster", "version"])
+    instance_args = (
```

Comment:
let's add a comment explaining why the behaviuor is differnt for 2.4.1 version

Reply:
Do we need this check here? Honestly I followed the code in previous `test_createami` function, there exists this line, but the integration test of `test_createami_post_install` will be enabled after 2.10. Should we skip this check?

Reply:
The reason is we added the instance-type parameter to the createami command in the [2.4.1 version of ParallelCluster](https://github.com/aws/aws-parallelcluster/releases/tag/v2.4.1), but given that this test is for 2.9.1+ I'd remove this check.

In any case if you copied it from the previous testwe could put most of this initialization code in common.

Reply:
Version prior to `2.4.1` wasn't supporting to specify the instance type to be used during buold. We can remove the logic

Reply:
Done